apps/workflows-dashboard/entrypoint.sh

```
@@ -0,0 +1,35 @@
+#!/usr/bin/env sh
+
+if [[ -n "$VITE_DOMAIN" ]]
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)